app/api/helpers/checksum.py

```diff
@@ -97,13 +97,17 @@ def __pad__(string_literal):
 def __unpad__(string_literal):
     return string_literal[0:-ord(string_literal[-1])]
 
-
 def __encode__(to_encode, iv, key):
+
+    # to_encode = to_encode.encode("UTF-8")
+    # iv = iv.encode("UTF-8")
+    # key = key.encode("UTF-8")
+
```

Comment:
Remove

Reply:
I thought to leave comments for below reference, Hence removing them would be a better option than converting them here ?

Reply:
You're repeating the encoding again and again, why not do it once and be done with it

Reply:
These three variables are used in three different functions occurring once, Hence changing them inside one scope won't work. That's why doing at the same time.

Reply:
OK. Then remove the comment